### ISSUE TITLE
Inconsistent swiping behavior for row layout manager

### ISSUE THREAD

**Issue by tonnyavery** [google/flexbox-layout#585]
- [x] I have searched [existing issues](https://github.com/google/flexbox-layout/issues) and confirmed this is not a duplicate

## Issues and steps to reproduce
Nesting recycler in with FlexboxLayoutManager and direction ROW in another recycler results in inconsistent touch events when swiping horizontally backwards.

## Expected behavior
Nested FlexboxLayoutManager should result in consistent swiping behavior.

## Version of the flexbox library
3.0.0

**Comment by kaiwalyakhasnis** [comment]
@AAverin were you able to find work around for this?

**Comment by tonnyavery** [comment]
@kaiwalyakhasnis No, for my use case I used LinearLayoutManager with some custom measurement implementation
